GETTING_STARTED.md

```diff
@@ -0,0 +1,63 @@
+# Getting Started as a Boutiques Developer
+
+This guide will help you set up your local environment for coding and testing Boutiques.
+
+## 0. Things you'll need
+- **A Linux system**. It _may_ be possible to develop Boutiques on Windows, but will likely cause many headaches.
+- **Root (sudo) access**. Unless your system already has everything you need installed, you'll need root permission to install new programs.

```

Comment:
I'd put this at the bottom of the list, since it only is required if other things are not installed. I develop on Compute Canada for Boutiques, and I certainly don't have root there :)